services/worker/tests/fixtures/hub.py

```diff
@@ -375,8 +375,8 @@ def create_parquet_and_dataset_info_response(dataset: str, data_type: Literal["c
     }
 
 
-CSV_PARQUET_SIZE = 1_865
-AUDIO_PARQUET_SIZE = 1_383
+CSV_PARQUET_SIZE = 1_866
+AUDIO_PARQUET_SIZE = 1_384
```

Comment:
Increasing size to have test running successful. Looks like pyarrow 11 increased the size of computed parquet file.